docs/definition-of-done.md

```diff
@@ -22,46 +26,74 @@ In most cases, you should be redirecting users to the start page of your service
+- each page works without JS
+- hidden context values have content, not raw message file properties
 
 ### Usability
 
-- provide appropriate context for screen readers where necessary
-- check you can complete this journey without a mouse
-- check you can complete this journey without a screen
-- check you can complete this journey on a mobile device
-- test each page is usable with **at least one** screen reader (see below)
-- test each page is usable with **at least one** screen magnifier (see below)
-- test each page is usable with **at least one** voice controller (see below)
+- complete the journey without a mouse
+- complete the journey without a screen
+- complete the journey on a mobile device
+- complete the journey with **at least one** screen reader (see below)
+- complete the journey with **at least one** screen magnifier (see below)
+- complete the journey with **at least one** voice controller (see below)
+
+When time is a factor, Windows users can test with NVDA. Mac users can test with VoiceOver and Voice Control, but you get better results on iOS.
+
+### Native OS settings
+
+- set system font size to 36px and check what breaks
+- make sure nothing disappears when inverted colours are on
+- check contrast levels are good enough in greyscale mode
+- ensure animation respects the OS reduce motion setting
 
 ### Screen readers
 
-- test the journey is successful with VoiceOver on macOS
-- test the journey is successful with VoiceOver on iOS
-- test the journey is successful with JAWS on Windows
-- test the journey is successful with NVDA on Windows (if you are on Linux try Orca)
+- if it looks like a heading, it should announce as a heading
+- screen reader users should be able to navigate to each element
+- everything should read out in a logical order
+- provide appropriate context for screen readers where necessary
+- actions aren’t over-explained with extra context for screen readers
 
 ### Screen magnifiers
 
-- test the journey is successful with ZoomText on Windows
-- test the journey is successful with Zoom enabled on macOS
-- test the journey is successful with Zoom enabled on iOS
+- everything should still be usable at 200%
+- everything should still be usable at 400%
 
 ### Voice control
 
-- test the journey is successful with Dragon
-- test the journey is successful with Voice Control
+- check each piece of functionality of the new feature is usable by voice recognition software
+- make sure any hidden context isn’t preventing voice activation from working
+
+### Orientation checks
+
+This applies to mobile devices and is covered by [WCAG 2.1 — 1.3.4 Orientation (Level A)](https://www.w3.org/WAI/WCAG21/quickref/#orientation). Check the following in portrait and landscape:
+
+- everything is correctly aligned
+- everything still aligns correctly when the screen scrolls
+- nothing important needs to be repositioned
+- screen width is fully utilised in landscape orientation
+- everything scrolls into view that should do
```

Comment:
Can we just point to the existing common testing methods rather than duplicate here?